pkg/apis/agones/v1/gameserver.go

```diff
@@ -284,6 +305,18 @@ func (gss *GameServerSpec) applyContainerDefaults() {
 	if len(gss.Template.Spec.Containers) == 1 {
 		gss.Container = gss.Template.Spec.Containers[0].Name
 	}
+
+	for idx, c := range gss.Template.Spec.Containers {
+		if c.TerminationMessagePolicy == "" {
+			gss.Template.Spec.Containers[idx].TerminationMessagePolicy = corev1.TerminationMessageFallbackToLogsOnError
+		}
+
+		if c.ImagePullPolicy == "" {
+			// TODO: Check if that is what the Agones team is expecting.
+			// Options: Always or IfNotPresent https://pkg.go.dev/k8s.io/api/core/v1@v0.15.11?tab=doc#PullPolicy
+			gss.Template.Spec.Containers[idx].ImagePullPolicy = corev1.PullIfNotPresent
```

Comment:
Which ImagePullPolicy is expected to be used by default?

Reply:
Whatever the default is for Kubernetes 👍